src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -10,37 +18,13 @@ import Breadcrumbs from "../../../components/Breadcrumbs"
 import Link from "../../../components/Link"
 import Translation from "../../../components/Translation"
 import PageMetadata from "../../../components/PageMetadata"
-import {
-  Content,
-  ListItem,
-  Page,
-} from "../../../components/SharedStyledComponents"
 
 // Utils
 import { translateMessageId } from "../../../utils/translations"
 import FeedbackCard from "../../../components/FeedbackCard"
 
-// Styles
-const HorizontalUl = styled.ul`
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
-
-  @media (max-width: ${(props) => props.theme.breakpoints.xl}) {
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-  }
-
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    grid-template-columns: 1fr 1fr 1fr;
-  }
-
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    grid-template-columns: 1fr 1fr;
-  }
-
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    grid-template-columns: 1fr;
-  }
-`
+const Content = (props: BoxProps) => <Box py={4} px={8} w="full" {...props} />
+const ContentHeading = (props: TextProps) => <Text as="h2" {...props} />
```

Comment:
Chakra provides a `Heading` component that should be used here.

This does mean that you will have to explicitly set the fontSizes, fontWeights, and lineHeights set in the [global styles](https://github.com/ethereum/ethereum-org-website/blob/ae5df7a0997720039d9124fb89c3a16c63af59b4/src/%40chakra-ui/gatsby-plugin/styles.ts#L67-L111) to override the defaults. But this will help with the transition to the new Design System and updating the theme config. 

By default, the `h2` element is rendered in `Heading` so that can be removed. And since the `lineHeight` value is the same for all headings, it can be defined here. :)
```suggestion
const ContentHeading = (props: HeadingProps) => <Heading lineHeight={1.4} {...props} />
```